libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackHelper.cs

```diff
@@ -107,5 +92,22 @@ public static bool VerifySignature(string secret, HttpRequest request, string bo
                 return hash == retrievedSignature;
             }
         }
+
+        public static NewSlackMessage GetChannelDataFromSlackEvent(dynamic slackEvent)
+        {
+            // Convert Slack timestamp format to DateTime
+            string[] splitString = slackEvent["event"].ts.ToString().Split('.');
```

Comment:
slackEvent['event'] -> should be inside a var so that you don't have to call all that every time you want it.

Reply:
This adds another dynamic variable. Anyway, I'll do the change.

Reply:
Could we use `var` for this new variable?

Reply:
@matiasroldan6 Also, add a description to this new method